src/app/assets/languages/en.json

```diff
@@ -61,6 +61,7 @@
     "undo-ignore" : "Show",
     "ignored-status" : "Hidden",
     "delete" : "Delete",
+    "available": "Available",
```

Comment:
@oraNod which should be the action name to set it back to available when the cache is degraded ?

Reply:
Commented in another section but "Make available" is the natural collocation imo.

Reply:
@dipguptaredhat please update the label